setup.py

```diff
@@ -38,7 +38,8 @@ def read_reqs(fname):
 long_description = read(os.path.join(os.path.dirname(__file__), "README.md"))
 requirements = read(os.path.join(os.path.dirname(__file__), "requirements.txt"))
 dev_reqs = read_reqs('requirements-dev.txt')
-extras_require = {"test": dev_reqs, "dev": dev_reqs}
+binder_reqs = read_reqs(os.path.join('binder', 'requirements.txt'))
```

Comment:
This needs to be
```suggestion
binder_reqs = read(os.path.join(os.path.dirname(__file__), 'binder', 'requirements.txt'))
```
-- I think that'll fix the build failure

Reply:
Should probably also update the dev_reqs to be the same pattern if you don't mind

Reply:
what are the odds I could use `pathlib` for this?

so

```
binder_reqs = Path(__file__).parent.joinpath('binder', 'requirements.txt').read_text()
```

Reply:
Should be fine. I still write 2/3 compatible code by default from years of habit